src/main/java/fr/insalyon/creatis/gasw/plugin/executor/dirac/execution/DiracJdlGenerator.java

```diff
@@ -115,10 +128,24 @@ public String generate(String scriptName, Map<String, String> envVariables) {
             velocity.put("site", site);
             velocity.put("bannedSite", bannedSites);
             velocity.put("tags", tags);
+            velocity.put("isMoteurliteEnabled", isMoteurliteEnabled);
+
+
+            // If MoteurLite is enabled, include these additional variables
+            if (isMoteurliteEnabled) {
```

Comment:
is it possible to take that from GaswConfiguration ?

Reply:
I don't see any engine related properties in gaswConfiguration